example_colabs/custom_data_overlay_demo.ipynb

```diff
@@ -1,310 +1,317 @@
-        "      )\n",
-        "      for (\n",
-        "          subgraph_profile\n",
-        "      ) in benchmark_profiling_proto.runtime_profile.subgraph_profiles:\n",
-        "        subgraph_profile_json = {}\n",
-        "        for op_profile in subgraph_profile.per_op_profiles:\n",
-        "          subgraph_profile_json.update(\n",
-        "              get_op_profile_json(op_profile, model_explorer_json_type)\n",
-        "          )\n",
-        "        output_json[subgraph_profile.subgraph_name][\n",
-        "            \"results\"\n",
-        "        ] = subgraph_profile_json\n",
-        "\n",
-        "        if model_explorer_json_type == _PER_OP_LATENCY_JSON_TYPE:\n",
-        "          output_json[subgraph_profile.subgraph_name][\"gradient\"] = [\n",
-        "              {\"stop\": 0, \"bgColor\": \"green\"},\n",
-        "              {\"stop\": 0.33, \"bgColor\": \"yellow\"},\n",
-        "              {\"stop\": 0.67, \"bgColor\": \"orange\"},\n",
-        "              {\"stop\": 1, \"bgColor\": \"red\"},\n",
-        "          ]\n",
-        "\n",
-        "  if output_path:\n",
-        "    with open(output_path, \"w\") as f:\n",
-        "      json.dump(output_json, f)\n",
-        "  else:\n",
-        "    print(json.dumps(output_json, indent=2))\n",
-        "\n",
-        "\n",
-        "CPU_PROFILING_JSON_PATH=\"/tmp/data/mv3-cpu-op-profile.json\"\n",
-        "\n",
-        "# Generate pure CPU per-op profiling JSON.\n",
-        "generate_model_explorer_json(\n",
-        "    [CPU_PROFILING_PROTO_PATH],\n",
-        "    CPU_PROFILING_JSON_PATH,\n",
-        "    _PER_OP_LATENCY_JSON_TYPE,\n",
-        ")\n",
-        "\n",
-        "# Download the XNNPACK per-op profiling JSON from storage.\n",
-        "!wget -nc https://storage.googleapis.com/tfweb/model-explorer-demo/mv3-xnnpack-op-profile.json\n",
-        "XNNPACK_PROFILING_JSON_PATH=\"/tmp/data/mv3-xnnpack-op-profile.json\""
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "oCgFNFP4WAzE"
-      },
-      "source": [
-        "# Visualize the model with per op latency"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "6SVyHrthM_hz"
-      },
-      "outputs": [],
-      "source": [
-        "import model_explorer\n",
-        "\n",
-        "config = model_explorer.config()\n",
-        "(config\n",
-        " .add_model_from_path(model_path)\n",
-        " .add_node_data_from_path(CPU_PROFILING_JSON_PATH)\n",
-        " .add_node_data_from_path(XNNPACK_PROFILING_JSON_PATH))\n",
-        "\n",
-        "model_explorer.visualize_from_config(config)"
-      ]
-    }
-  ],
-  "metadata": {
-    "colab": {
-      "provenance": []
-    },
-    "kernelspec": {
-      "display_name": "Python 3",
-      "name": "python3"
-    },
-    "language_info": {
-      "name": "python"
-    }
+ "cells": [
```

Comment:
Just curious, what are the changes in these two notebooks? (hard to tell from github diff)

Reply:
It did some indentation formatting of the colab scripts (changing from 4 spaces to 2 spaces).